tests/components/application-connector/k8s/gateway-test/templates/basic-cases.yml

```diff
@@ -0,0 +1,42 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: test-app
```

Comment:
Shouldn't this Application be named `positive-cases-app` or something like this? Also maybe the file could be renamed to `positive-cases-app.yml`. What do you think? Then the failures would be `negative...` or `failing-cases-app`

Reply:
Considering the response codes can be stored in the AppCR, we could just have basic cases and merge failing (with wrong secrets) cases into this - the only difference in tests is the response code

Reply:
Ok, it's fine. I was biased by the previous idea :D